Luminescent/src/astechzgo/luminescent/keypress/KeyPressUtils.java

```diff
@@ -32,23 +34,35 @@ public static void checkUtils() {
 				KeyboardUtils.resetKeys();
 			}
 		}
-		if(KeyboardUtils.isKeyDown(Constants.KEYS_UTIL_SCREENSHOT)) {
-			File dir = newFile("screenshots/");
-		
-			if(!dir.exists() || !dir.isDirectory()) {
-				dir.mkdir();
-			}
-		
-			SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
-			Date dt = new Date();
-			String S = sdf.format(dt);
-			
-			try {
-				DisplayUtils.takeScreenshot(newFile("screenshots/" + S + ".png"));
-			} catch (Exception e) {
-				e.printStackTrace();
+		if(KeyboardUtils.isKeyDown(Constants.KEYS_UTIL_SCREENSHOT)) {			
+			if(!takingScreenshot) {
+				File dir = newFile("screenshots/");
+				
+				if(!dir.exists() || !dir.isDirectory()) {
+					dir.mkdir();
+				}
+				
+				SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd_HH.mm.ss");
+				Date dt = new Date();
+				String S = sdf.format(dt);
+				
+				try {
+					DisplayUtils.takeScreenshot(newFile("screenshots/" + S + ".png"));
+				} catch (Exception e) {
+					e.printStackTrace();
+				}
+				
+				if(Luminescent.SOUND_MANAGER.getSoundSystem().playing("keys.util.screenshot.CameraClick"))
+					Luminescent.SOUND_MANAGER.getSoundSystem().stop("keys.util.screenshot.CameraClick");
+				
+				Luminescent.SOUND_MANAGER.getSoundSystem().play("keys.util.screenshot.CameraClick");
+				
+				takingScreenshot = true;
 			}
 		}
+		else if(takingScreenshot) {
```

Comment:
This seems kind of hacky to me.  It also doesn't work for other things this may apply for, like pause in the future.  I may add a `isPressedOnce()` method like as mentioned in #110.